syntax/braces.go

```diff
@@ -107,11 +107,12 @@ func SplitBraces(word *Word) *Word {
 				broken := false
 				for i, elem := range br.Elems[:2] {
 					val := elem.Lit()
-					if _, err := strconv.Atoi(val); err == nil {
-					} else if len(val) == 1 &&
-						'a' <= val[0] && val[0] <= 'z' {
+					switch _, err := strconv.Atoi(val); {
```

Comment:
I don't think this change is worth it; the if-elseif chain only had three elements, and this new define-switch is more complex than needed.

Reply:
Maybe then return the if-else, but remove the empty block?
```go
if err == nil
```
?

Reply:
Maybe - feel free to take a stab at that.

Reply:
OK, it was a dumb question. It means if err == nil do nothing/

Reply:
well, if there was no error, there's nothing else to do. if there was an error, we move on to the other cases.

Reply:
Done in e336613e7883ab4a30afe31f6d84c28752fa2f75.